config/routes.rb

```diff
@@ -7,4 +7,5 @@
   end
 
   post '/webhooks/bolt', to: '/solidus_bolt/webhooks#update'
+  post '/accounts/bolt', to: '/solidus_bolt/accounts#create'
```

Comment:
This should be under `/api`, doesn't it?

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/107/commits/cbad318944309509197f3931924794104865de47